test/credentials_test.dart

```diff
@@ -22,13 +22,13 @@ import 'package:test/test.dart' hide test, throws;
 import '../lib/realm.dart';
 import 'test.dart';
 
-Future<User> retryLogin(int retries, Future<User> Function(Credentials credentials) doFunction, Credentials credentials) async {
+Future<User> loginWithRetry(App app, Credentials credentials, int retryCount) async {
```

Comment:
It's a good idea to make the `retryCount` optional here or even hardcode it to some value. The only reason to have a retryCount is to make sure we don't retry forever, but there's no reason for consumers of the method to control the value.

Reply:
Done